packages/shorebird_code_push_protocol/lib/src/messages/create_release/create_release_request.dart

```diff
@@ -27,6 +28,10 @@ class CreateReleaseRequest {
   /// The release version.
   final String version;
 
+  /// The Flutter revision used to create the release.
+  /// This is nullable for backward compatibility.
```

Comment:
I'd maybe add a TODO here to make this non-optional at some arbitrary point in the future.

Reply:
Will do that in the next PR 👍